deploy/common/deploy.sh

```diff
@@ -296,3 +296,23 @@ EOF
 }
 EOF
 }
+
+function setup_internal_sso() {
+    local LOCAL_API_ENDPOINT="$1"
+    local LOCAL_CLIENT_SECRET="$2"
+	echo "Setting up Dev Internal SSO login"
+
+    roxctl declarative-config create auth-provider oidc \
+        --secret=declarative-configurations \
+        --namespace=stackrox \
+        --name="Internal-SSO" \
+        --ui-endpoint="${LOCAL_API_ENDPOINT}" \
+        --minimum-access-role=Admin \
+        --extra-ui-endpoints=localhost:8000 \
+        --extra-ui-endpoints=localhost:3000 \
```

Comment:
Probably would be good to have the following UI endpoint as well: `localhost:8443`.
Reason for this is the default value for roxctl's central endpoint is `localhost:8443` and if you port-forward manually typically you do `k port-forward -n stackrox svc central 8443:8443`.